CODEOWNERS

```diff
@@ -353,6 +353,7 @@
 /dts/arm/armv*m.dtsi                      @galak @ioannisg
 /dts/arm/armv7-r.dtsi                     @bbolen @stephanosio
 /dts/arm/armv8-a.dtsi                     @carlocaione
+/dts/arm/armv8-r.dtsi                     @povergoing
```

Comment:
only this one? Add also all the entries for `cortex_r/aarch64` files.

Reply:
No problem, I will add all relevant entries